rclcpp/include/rclcpp/callback_group.hpp

```diff
@@ -94,6 +94,20 @@ class CallbackGroup
     return _find_ptrs_if_impl<rclcpp::Waitable, Function>(func, waitable_ptrs_);
   }
 
+   /**
+   * Returns size of callback group which includes
+   * subscriptions, timers, clients, services, and
+   * waitable
+   *
+   * \return size of callback group
+   */
+  size_t
+  size() const
+  {
+    return subscription_ptrs_.size() + timer_ptrs_.size() +
+           client_ptrs_.size() + service_ptrs_.size() + waitable_ptrs_.size();
```

Comment:
Make sure you run the tests, I'm a bit surprised this passed the linters, but maybe.